exercises/space-age/src/example.h

```diff
@@ -1,18 +1,18 @@
 #ifndef SPACE_AGE_H
 #define SPACE_AGE_H
 
-/* Number of seconds in a Earth year */
-#define EARTH 31557600
+typedef enum planets {
+   mercury = 0,
+   venus,
+   earth,
+   mars,
+   jupiter,
+   saturn,
+   uranus,
+   neptune,
+   max_planet = neptune
```

Comment:
Sorry, I was pretty unclear here. The `= neptune` part is unnecessary because `max_planet` will default to `neptune + 1` which you can use for a `for` limit without making the enum more difficult to maintain by needing to babysit the max definition as items are added.